### PR TITLE
Message of the day post request replaced

### PR DESCRIPTION
**Pull request by inaod568**
This requires a follow up task: https://scilifelab.atlassian.net/browse/DDS-1270?atlOrigin=eyJpIjoiY2EwY2I5NDNkM2RmNGQ4OWEzODMyNmQ2NWM4ZGZhYTciLCJwIjoiaiJ9

Before submitting a PR to the `dev` branch:
- [x] Tests passing
- [x] Black formatting
- [x] Rebase/merge the `dev` branch

Additional checks before submitting a PR to the `master` branch:
- Change version in `dds_cli/version.py` (?)

### CODE REVIEW

File: dds_cli/motd_manager.py [hunk 1]
```diff
@@ -62,43 +62,13 @@ def __init__(
...
-        LOG.info("A new MOTD was added to the database")
+        response_json = dds_cli.utils.perform_request(
+            endpoint=DDSEndpoint.ADD_NEW_MOTD,
+            headers=self.token,
+            method="post",
+            json={"message": message},
+            error_message="Failed adding a new MOTD",
+        )
+
+        message = response_json.get("message", "A new MOTD was added to the database")
```
Comment (valyo) [inline dds_cli/motd_manager.py]: A quick test shows that response_json is None so this get fails. This must be my motd endpoint not returning anything, right?

**Review by valyo — CHANGES_REQUESTED** [review]
We either skip the response_json message now, or we open a task to fix dds_web accordingly

File: dds_cli/motd_manager.py (hunk 1 above)
Comment (inaod568) [reply dds_cli/motd_manager.py]: Yep, fixed

**Review by valyo — APPROVED** [review]
no stack backtrace now :)

**Pull request merged**